frontend/packages/kubevirt-plugin/src/selectors/event/filters.ts

```diff
@@ -10,18 +10,33 @@ import {
 import { VIRT_LAUNCHER_POD_PREFIX } from '../../constants/vm';
 import { VMILikeEntityKind } from '../../types/vmLike';
 
-type EventFilterFunction = (src: EventInvolvedObject) => boolean;
+type EventInvolvedObjectWithTimestamp = EventInvolvedObject & { timestamp?: string };
+
+type EventFilterFunction = (src: EventInvolvedObjectWithTimestamp) => boolean;
 
 const vmEventFilter = (vm: VMILikeEntityKind): EventFilterFunction => ({
   kind,
   namespace,
   name,
+  uid,
 }) => {
   return (
-    kind === VirtualMachineModel.kind && name === getName(vm) && namespace === getNamespace(vm)
+    kind === VirtualMachineModel.kind &&
+    name === getName(vm) &&
```

Comment:
i think we dont need to check name and namespace anymore, `uid` is enough

Reply:
done